### ISSUE TITLE
Missing profile image on second slide

### ISSUE THREAD

**Issue by spcanelon** [LizHareDogs/iwdba2021#15]
Hey @LizHareDogs, just a heads up that when I was browsing through your slides I noticed your profile picture was missing from slide 2. It might be because there is a typo in the html code that inserts the JPG: the alt text attribute is missing the "a" in "alt":

https://github.com/LizHareDogs/iwdba2021/blob/34485998c894bb2743e5b1cb960399f4ba77da61/index.Rmd#L141-L145

No issues with the profile picture on the last slide though, that one shows up fine!

**Comment by LizHareDogs** [comment]
Thanks so much @spcanelon ! It was also the filename extension not being capitalized🤯
